### ISSUE TITLE
Question about list trustedmac

### ISSUE THREAD

**Issue by cormacstrain** [openNDS/openNDS#504]
Hi Rob

I need advice from you once again

Im using a printer that prints from a website to my printer on the network using a plugin.  With OpenNDS off, everything works fine. When I turn OpenDNS on - as expected it doesn't work as I haven't allowed access to whatever domain(s) the communication between the plugin and my printer may require.  

I cant really track down what the issue as I can see no mention of anything bar sending data to my network, through the site etc that I am connecting to - all of which are working and accessible, is so I tried to see if it was an authentification issue by putting the printer's Mac address in the config with 'list trustedmac', but to no avail.

Is this because I am using list trustedmac incorrectly, or would it mean that its not an authentification issue and theres some other reason?   

TL;DR why is the printer plugin running on the site defined as option fasremotefqdn not connecting to my home network whenever opennds is running.  It works fine through the same router with OpenDNS off.

many thanks
Cormac

**Comment by bluewavenet** [comment]
@cormacstrain 

> Im using a printer that prints from a website to my printer on the network using a plugin. 

I don't really follow what you are trying to do. Are you saying a remote Internet based web site wants to come in to a printer on your lan?

For this you would need a public ip address and port forwarding and without this would not work even without opennds running -  but you say it works....

So you need to look in detail what traffic goes from where to where, what ip addresses, port numbers etc.

What is the plugin? Is it running on the web site or somewhere else?

**Comment by cormacstrain** [comment]
Hi Rob!

What I have is a Star MCPrint 2 printer which uses a wordpress 'star cloudprnt woocommerce' plugin to connect the printer to the website.  Its an ecommerce site so anything purchased on the site prints a receipt on the printer the same router as opennds.  The plugin runs on the site, and in the printer config (on my local network), I put in the url to the site - thats how the printer and plugin link up and communicate.

Ive went through the  'star cloudprnt woocommerce'  plugins files and I cant see any reference to any external IP, domain or port.  The site the plugin is running from is the domain used in the fasremotefqdn.  I cant find any reference to the plugin or printer trying to communicate to anything other than the domain and the router itself.

Thats why I was wondering if I could use list trustedmac with the printers mac address and therefore it would have access to anywhere it needs.  Would that be correct?

**Comment by bluewavenet** [comment]
@cormacstrain 
Yes, trustedmac should give free access.

Check if you configured it correctly by doing:
`ndsctl status`
Trusted macs are listed at the end.

**Comment by cormacstrain** [comment]
thanks Rob!  that answers my question. 

cheers

Cormac

**Comment by cormacstrain** [comment]
have it working now
